pkg/virt-controller/services/template.go

```diff
@@ -360,6 +361,14 @@ func (t *templateService) RenderLaunchManifest(vmi *v1.VirtualMachineInstance) (
 	return t.renderLaunchManifest(vmi, false)
 }
 
+func (t *templateService) SetCpuArch(arch string) {
+	t.setCpuArch(arch)
+}
+
+func (t *templateService) setCpuArch(arch string) {
+	t.cpuArch = arch
```

Comment:
Is this function redundant? Can't it be collapsed into `SetCPUArch` (btw. normally `CPU` in golang vs `Cpu` in e.g. java)?

Reply:
> collapse

Yes, it can put into SetCPUArch, I thought it is some kind of coding format, like `RenderLaunchManifest`. So I just follow it.